code42.json

```diff
@@ -75,26 +93,28 @@
                     "allow_list": true
                 },
                 "start_time": {
-                    "data_type": "numeric",
-                    "order": 1,
-                    "description": "Start of time range, in epoch time (milliseconds)",
-                    "verbose": "If not specified, the default is past 10 days"
+                    "description": "The start time to use in the initial poll in %Y-%m-%d %H:%M:%S format (defaults to 30 days back)",
```

Comment:
They are strings since they are input from the configuration page, but what we store / checkpoint is a float.